### PR TITLE
Add diagrams to PEP-001

### PR DESCRIPTION
**Pull request by carolynvs**
Add diagrams to PEP-0001 to clarify states and their transitions.

### CODE REVIEW

File: pep/001-initial-pep-process.md [hunk 1]
```diff
@@ -317,39 +319,46 @@ This is the happy path, where a PEP is accepted.
...
-  explain the rejection in the active pull request and ultimately close the pull
-  request unmerged. The reviewer submits a pull request updating the status to
-  `rejected` and reiterate the rejection, including enough context that someone
-  can understand it without going back to old pull requests. The status is
-  **REJECTED**.
+* **REJECTED**: The proposal has been rejected. The PEP may be rejected at
+  certain points by the reviewer. Reviewer should explain the rejection in the
+  active pull request and ultimately close the pull request unmerged. The
+  reviewer submits a pull request updating the status to `rejected` and
+  reiterate the rejection, including enough context that someone can understand
```
Comment (vdice) [inline pep/001-initial-pep-process.md:339]:
```suggestion
  reiterates the rejection, including enough context that someone can understand
```

**Review by vdice — APPROVED** [review]
I see `images/workflow.pptx` was added as well, but there doesn't seem to be a link to it in the PEP.  Perhaps intended as an auxiliary resource for future reference?

**Comment by carolynvs** [comment]
Correct, images/workflow.pptx is what I used to generate the diagram images. It's included so that anyone can edit them later.

**Comment by iennae** [comment]
I LIKE them! Great job! I think it adds clarity to the process. The one improvement I could possibly see (And totally not required) is associating the number in the steps below the diagram on the diagram itself so as you read the steps you can look at the diagram and find the number.

**Review by iennae — APPROVED** [review]
LGTM! Thank you :D

**Comment by iennae** [comment]
I also like the update in the states descriptions and having the clear state to what it means definition! Great job!

**Comment by carolynvs** [comment]
I'm not sure adding the numbers from the markdown into the image wouldn't cause confusion. Not all steps are represented in the diagram, so they would skip, e.g. 3 - assign reviewers isn't represented. I'm going to merge as-is and if people are confused by what's there now we can reassess.

**Pull request merged**